src/components/side-panel.js

```diff
@@ -161,7 +163,10 @@ export default function SidePanelFactory(
       this.props.uiStateActions.toggleModal(SAVE_MAP_ID);
     };
 
-    _onClickShareMap = () => this.props.uiStateActions.toggleModal(SHARE_MAP_ID);
+    _onClickShareMap = () => {
+      this.props.providerActions.resetProviderStatus();
```

Comment:
why are we resetting the status when we open the share modal dialog?

we should reset the provider status whenever we complete a sharing action,e .g. sharing successful

Reply:
I applied it when opening to ensure that there is no error pending error in the state from any source.
In any case, I think that it must be reset after closing the save or share dialog in order to show the messages to user; but I'm afraid of showing other pending error message that could not be cleared before.

Reply:
`resetProviderStatus` is called when Share map modal is closed. it sets 
```
  isProviderLoading: false,
  providerError: null,
  successInfo: {}
```

It's not very pretty but works. If you need to, we should probably call reset status when close `LoadDataModal` `SaveMapMadal` and `ShareMapModal`

Reply:
Ok, I'll apply it this way.

Reply:
I've applied as you advised and it looks way better, thanks! Is it OK for you?